addons/mail/static/src/scss/emojis.scss

```diff
@@ -1,12 +1,15 @@
+// General variable
+$o-mail-emoji-height: 25px;
```

Comment:
Good idea to use a variable. 

Could you convert the pixel amount into rem too ? 

Thanks !

Reply:
@tivisse Done !